packages/superset-ui-chart/src/components/ChartDataProvider.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint react/sort-comp: 'off' */
 import React, { ReactNode } from 'react';
-import { SupersetClientInterface, RequestConfig } from '../../../superset-ui-connection/src/types';
+import { SupersetClientInterface, RequestConfig } from '@superset-ui/connection/lib/types';
```

Comment:
Would this work?
```js
import { SupersetClientInterface, RequestConfig } from '@superset-ui/connection';
```

Reply:
I'll try this, it's giving a TS error locally / can't find the declaration file 🤔